cmd/skopeo/proxy.go

```diff
@@ -173,6 +174,12 @@ type proxyHandler struct {
 	activePipes map[uint32]*activePipe
 }
 
+type blobInfo struct {
```

Comment:
Following precedent, please document this type — at least enough to say that is used in the return value of `LayerInfoJSON`.